ceph-lttng/README.md

```diff
@@ -0,0 +1,12 @@
+Builds an image with Ceph compiled with LTTnG. The image also has LTTnG tools
+for starting, stopping, and viewing the tracepoints.
+
+===================================================
+Quickstart
+===================================================
+
+Build Ceph with LTTnG and put it in an image:
+
+```bash
+CEPH_SRC=~/ceph ./build
+```
```

Comment:
Were you able to generate a daemon image with your changes? Looks like we're missing the modifications to `ceph-base/scripts/generate_daemon_image`, were you thinking in having those as a separate changeset?